mappings/1fd25ea9b09b4b666ff927c409a5a134661d1f7288bddbb84d226b2542696b654d696b65.json

```diff
@@ -0,0 +1,64 @@
+{
+    "subject": "1fd25ea9b09b4b666ff927c409a5a134661d1f7288bddbb84d226b2542696b654d696b65",
+    "url": {
+        "sequenceNumber": 0,
+        "value": "https://bigmike2024.online",
```

Comment:
Hi @Ganymede88,

seems like your website is not working. You can update the submission once it is running, url value is optional. Please, update the value with a valid page or delete the value.

Thanks!

Reply:
Hi @VladislavKudrin 
Thank you for the response. 
https://bigmike2024.online is up and running  :)
Let me know if you need anything else thanks.
-G